src/main/resources/data.sql

```diff
@@ -83,568 +86,51 @@ insert into user (id, first_name, middle_name, last_name, phone_number) values
-('f334199f-5f22-4ce5-b980-d2df4a1085c0', 'Kellan', 'Preston', 'Andrews', '2461744165'),
-('f375aa7f-e081-49d0-870f-eebcd2b9d399', 'Walter', 'Martin', 'Murray', '6160371033'),
-('f3c5dc29-1089-4887-b73c-0ef0c057f24f', 'Derek', null, 'Scott', '4714751567'),
-('f46900f0-8331-4653-a209-9618d68f059c', 'Garry', 'Arthur', 'Mason', '5560531623'),
-('f46ee9b9-2b26-45b6-9494-dfeb21d7324f', 'Jared', null, 'Andrews', '1736221512'),
-('f479cdaf-2163-4ef3-aa01-e5235f16a462', 'Deanna', null, 'Brown', '2644013341'),
-('f4823101-8425-4b5d-9024-575ea4e8e703', 'Abraham', null, 'Higgins', '5732643020'),
-('f4cc8426-2ed3-4c16-98ce-96605f5324eb', 'Frederick', null, 'Davis', '3344434621'),
-('f5164fd3-b259-4d36-a7cb-5d37d5d3ab07', 'Kellan', null, 'Casey', '5044561015'),
-('f52f40de-6cc3-45d9-b4dd-8d743391ba8e', 'Frederick', null, 'Kelley', '7441421171'),
-('f579b2a5-6d58-4a4d-b536-5e8f835e6fc7', 'Sofia', null, 'Farrell', '5642647625'),
-('f62bcea8-61c1-4f8d-ab9e-e5ac28a65013', 'Chelsea', null, 'Foster', '6221501645'),
-('f69538af-5a9b-4130-b708-667cd588035b', 'Alberta', null, 'Howard', '5524655527'),
-('f6cd5842-da01-4e59-b676-5ebc7e4a2693', 'Lydia', 'Rebecca', 'Harris', '5134251606'),
-('f7025046-5199-45f6-97bb-92a48ec0eb82', 'James', 'Reed', 'Morrison', '3775512223'),
-('f7f4a515-7f64-4718-8bb1-0e2f7b04465b', 'Fenton', 'Ashton', 'Wright', '1324044616'),
-('f8160123-1766-41c7-8056-2acbfd3c4f2a', 'Victor', null, 'Taylor', '3426104323'),
-('f8730cc6-a89b-40ca-94b2-ff4f7b3e0044', 'Catherine', null, 'Smith', '7653356573'),
-('f8bbbc7a-867b-4e8d-a2a5-8b7c07a4589c', 'Roland', null, 'Ross', '4655666462'),
-('f8f76b77-7007-47a5-9c50-5dfc26f59eb5', 'Roland', 'Alen', 'Murray', '5245223310'),
-('f9096b14-daab-430f-835b-2f0d651e3559', 'Arianna', 'Belinda', 'West', '4323425442'),
-('f9190ad7-930a-47d3-a6ce-84376082ff0b', 'Sam', null, 'Thompson', '1431023164'),
-('f97177e1-5fe1-4327-9978-9e9d48b23338', 'Carlos', null, 'Allen', '3111463363'),
-('fa98771c-4b54-4552-9c51-2d26ec988fa8', 'Alen', null, 'Williams', '5356263502'),
-('fae16dca-950f-441d-bc35-415d92f2e939', 'April', null, 'Turner', '1134214353'),
-('fb4d9aba-3c4b-45f2-8f94-8ad144c7dff7', 'Frederick', null, 'Fowler', '2273766242'),
-('fb89e37f-96b8-4d9a-b4e1-b51c592a6c1c', 'Hailey', null, 'Edwards', '2652441101'),
-('fc0e0a12-6679-4b44-8586-9d8eee063629', 'Alexia', 'Cadie', 'Carter', '3152614653'),
-('fcee9b60-6a3f-4fae-b7b3-27bc70c68b6f', 'Daisy', 'Adele', 'Williams', '2264357140'),
-('fd10b53c-a428-49a1-9628-f96f5dcaaed6', 'Carl', null, 'Thomas', '5455464561'),
-('fd152dd9-6fae-4cb2-a041-02162f200678', 'Brooke', 'Aiden', 'Watson', '2535073126'),
-('fd6d21f6-f1c2-473d-8ed7-f3f9c7550cc9', 'Elian', 'Cherry', 'Gibson', '3226211765'),
-('fd9e22ef-20f1-4e1b-9f9e-3139fd13fa85', 'Ashton', 'Carina', 'Foster', '4653236364');
+('0fa26435-0d04-4afe-b7aa-9a86d71b33a4', 'Clark', null, 'West', '5221356551');
+--('107f2920-9b79-4112-802c-47bc145acd56', 'Carlos', 'Lyndon', 'Perkins', '1136532503'),
```

Comment:
Like that you scrapped some of these as they didn't add a lot of value for the tests.  Commenting them is confuses me, is there a reason for the comment over the delete?

Reply:
I deleted because I felt that having so many wasn't all that helpful.
I could of commented some out as well, but it still makes the file and navigating the file a bit cumbersome.